Aztec/Classes/TextKit/LayoutManager.swift

```diff
@@ -4,8 +4,17 @@ import QuartzCore
 
 class LayoutManager: NSLayoutManager {
 
+    var blockquoteBorderColor: UIColor = UIColor(red: 0.52, green: 0.65, blue: 0.73, alpha: 1.0)
+    var blockquoteBackgroundColor = UIColor(red: 0.91, green: 0.94, blue: 0.95, alpha: 1.0)
```

Comment:
These should be set in the `LayoutManager` initializer call IMHO, in order to force the owner app to set a default value.

[The method that initializes the `LayoutManager` instance](https://github.com/wordpress-mobile/WordPress-Aztec-iOS/blob/try/refactor_layoutmanager/Aztec/Classes/TextKit/TextView.swift#L56) already sets a few other defaults, so it would be in-line with the rest of the code.